File: feature-promo/src/main/java/com/levnovikov/feature_promo/promo_list/dependency/PromoListDependency.java
package com.levnovikov.feature_promo.promo_list.dependency;

import com.levnovikov.system_base.node_state.ActivityState;

/**
 * Author: lev.novikov
 * Date: 4/1/18.
 */

public interface PromoListDependency {

    OnPromoSelectedListener onPromoSelectedListener();
    ActivityState activityState();
}

File: feature-promo/src/main/java/com/levnovikov/feature_promo/promo_list/dependency/PromoListDependency.kt
package com.levnovikov.feature_promo.promo_list.dependency

import com.levnovikov.system_base.base_di.ActivityComponent

/**
 * Author: lev.novikov
 * Date: 4/1/18.
 */

interface PromoListDependency : ActivityComponent {

    fun onPromoSelectedListener(): OnPromoSelectedListener
}
